src/routes/home.js

```diff
@@ -20,6 +20,19 @@ function prepareHome(data) {
   return result;
 }
 
+function prepareCalendar(data) {
+  const result = JSON.parse(data);
+
+  result.calendar.forEach(element => {
+    const weekday = moment(element.date, "DD.MM.YYYY")
+      .format("dddd")
+      .toLowerCase();
+    element.dayicon = `img/calendar/${weekday}.png`;
+  });
+
+  return result;
+}
+
```

Comment:
This here seems to be false. Please, consult @starmaerker :)

Reply:
I think it should not be there at all.

Reply:
@pheeria @starmaerker Interesting...I guess this happened when I used 'git pull origin master'. I now copied home.js from the current master and replaced the file. Thanks for noticing!